common/views/components/PageLayoutDeprecated/PageLayoutDeprecated.js

```diff
@@ -8,6 +8,7 @@ import Head from 'next/head';
 import convertUrlToString from '../../../utils/convert-url-to-string';
 // $FlowFixMe (tsx)
 import Header from '../Header/Header';
+import HeaderPrototype from '../Header/HeaderPrototype';
```

Comment:
You need a `FlowFixMe`
```suggestion
// $FlowFixMe
import HeaderPrototype from '../Header/HeaderPrototype';
```